assets/js/edit.js

```diff
@@ -54,330 +55,503 @@ function useFeaturedImage() {
-							{featuredImageUrl && (
-								<img src={featuredImageUrl} alt="Cover Image" />
+						</MediaUploadCheck>
+						{featuredImageUrl && (
+							<Button isLink isDestructive onClick={removeFeaturedImage}>{__('Delete Cover Art', 'simple-podcasting')}</Button>
+						)}
+					</PanelRow>
+				</PanelBody>
+			</InspectorControls>
+			<div className="wp-block-podcasting-podcast-outer">
+				{src ? (
+					<>
+						<div className="wp-block-podcasting-podcast__container">
+							{showImage && displayArt && (
+								<div className="wp-block-podcasting-podcast__show-art">
+									<div className="wp-block-podcasting-podcast__image">
+										<img
+											src={showImage}
+											alt={showName}
+										/>
+									</div>
+								</div>
 							)}
 
-							<MediaUploadCheck>
-								<MediaUpload
-									onSelect={onUpdateImage}
-									allowedTypes={['image']}
-									render={({ open }) => (
-										<Button isSecondary onClick={open}>
-											{featuredImageUrl ?
-											__('Replace Cover Art', 'simple-podcasting')
-											:
-											__('Select Cover Art', 'simple-podcasting')
-											}
-										</Button>
+							<div className="wp-block-podcasting-podcast__details">
+
+								{displayEpisodeTitle && postTitle && (
+									<h3 className="wp-block-podcasting-podcast__show-title">
+										{displayEpisodeNumber && episodeNumber && (
+											<span>
+												{episodeNumber}.
+											</span>
+										)}
+										{postTitle}
+									</h3>
+								)}
+
+								<div className="wp-block-podcasting-podcast__show-details">
+									{displayShowTitle && (
+										<span className="wp-block-podcasting-podcast__title">
+											{showName}
+										</span>
 									)}
-									value={featuredImageId}
-								/>
-							</MediaUploadCheck>
-							{featuredImageUrl && (
-								<Button isLink isDestructive onClick={removeFeaturedImage}>{__('Delete Cover Art', 'simple-podcasting')}</Button>
-							)}
-						</PanelRow>
-					</PanelBody>
-				</InspectorControls>
-				<div className={className}>
-					{src ? (
+									{displaySeasonNumber && seasonNumber && (
+										<span className="wp-block-podcasting-podcast__season">
+											{__(
+												'Season: ',
+												'simple-podcasting'
+											)}
+											{seasonNumber}
+										</span>
+									)}
+									{displayEpisodeNumber && episodeNumber && (
+										<span className="wp-block-podcasting-podcast__episode">
+											{__('Episode: ', 'simple-podcasting')}
+											{episodeNumber}
+										</span>
+									)}
+								</div>
+
+								<div className="wp-block-podcasting-podcast__show-details">
+									{displayDuration && duration && (
+										<span className="wp-block-podcasting-podcast__duration">
+											{__('Listen Time: ', 'simple-podcasting')}
+											{duration}
+										</span>
+									)}
+									{displayEpisodeType && (
```

Comment:
```suggestion
									{displayEpisodeType && (episodeType !== 'none') && (
```

Code untested but the idea is to prevent the following. You may wish to put the test elsewhere. 

![Screen Shot 2024-05-08 at 10 02 18 am](https://github.com/10up/simple-podcasting/assets/519727/1a739e8a-26d8-4db0-86dc-70a88ff02c63)